tools/kmod/sof_remove.sh

```diff
@@ -55,6 +55,7 @@ remove_module snd_soc_sof_sdw
 remove_module snd_soc_ehl_rt5660
 remove_module snd_sof_ipc_test
 remove_module snd_sof_probes
+remove_module snd_sof_intel_client
```

Comment:
don't we also need remove_module snd_sof_client?

Reply:
by Makefile change, the module name is snd_sof_intel_client

`rmmod: ERROR: Module snd_sof is in use by: snd_sof_intel_client`

Reply:
snd_sof_intel_client uses snd_sof_client in the latest PR, see e.g. https://github.com/thesofproject/linux/pull/2264/commits/e0333e48f760e08e95635960fe4ed4072b37e790

Reply:
thanks, I tested in APL UP2. Let me check how that passed

Reply:
@plbossart I checked the Makefile again. there is no snd_sof_client module. sof-client.c is part of snd-soc.ko. I remember also it was separate module in between version, but not anymore. Do I miss anything?

Reply:
@plbossart the way the Makefile is set up now sof-client.c will be built into the snd-sof module. let me make a minor change to the Makefile to make it a separate module

Reply:
but is there a particular reason why it should be a separate module?

Reply:
> but is there a particular reason why it should be a separate module?

you export a namespace, it'd be odd if that namespace was part of another module which defines another namespace.